crates/vim/src/state.rs

```diff
@@ -193,6 +193,14 @@ impl EditorState {
         self.operator_stack.last().cloned()
     }
 
+    pub fn current_operators_description(&self) -> String {
+        self.operator_stack
+            .iter()
+            .map(|item| item.id())
+            .collect::<Vec<_>>()
+            .join("")
+    }
+
```

Comment:
I'd personally move the contents of this method to the `ModeIndicator`, since it's a "view"-concern to me.

Reply:
That makes sense, thanks for the reminder